module/category-tree/src/Domain/Event/CategoryTreeNameChangedEvent.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\CategoryTree\Domain\Event;
+
+use Ergonode\Core\Domain\ValueObject\TranslatableString;
+use Ergonode\EventSourcing\Infrastructure\DomainEventInterface;
+use JMS\Serializer\Annotation as JMS;
+
+/**
+ */
+class CategoryTreeNameChangedEvent implements DomainEventInterface
```

Comment:
I think we have AbstractTranslatableChangedEvent?

Reply:
I couldn't find it